source/talk/SConscript

```diff
@@ -0,0 +1,6 @@
+Import('*')
+
+target = ['#output/talk/talk_slides.pdf', '#temp/talk_slides_slides.pdf', '#output/talk/talk_slides_slides.pdf']
+source = ['#source/talk/talk_slides.lyx']
+
+env.Lyx(target, source, HANDOUT_SFIX='_slides')
```

Comment:
```suggestion
env.Lyx(target, source)
```

I like having the "suffix" functionality in the builder but since it's pretty safe to use the builder without it, maybe we can omit the suffix declaration from this "demo"?